packages/waku/src/lib/renderers/rsc-renderer.ts

```diff
@@ -189,9 +189,9 @@ export async function renderRsc(
     ) {
       // XXX This doesn't support streaming unlike busboy
       const formData = parseFormData(bodyStr, contentType);
-      args = await decodeReply(formData);
+      args = await decodeReply(formData, bundlerConfig);
```

Comment:
I actually wonder why this works. This is basically resolving client entry. Can it be used for server entry?

Reply:
I think it's also resolving the server entry, yes

Reply:
check react test case, they have cases to load server modules

https://github.com/facebook/react/blob/29552c7907230222acd3f2c586784d24f9da6200/packages/react-server-dom-webpack/src/__tests__/ReactFlightDOMReply-test.js#L43

https://github.com/facebook/react/blob/29552c7907230222acd3f2c586784d24f9da6200/packages/react-server-dom-webpack/src/__tests__/utils/WebpackMock.js#L130

Reply:
In `bundlerConfig`, we use `resolveClientEntry`, so even if it works, it's confusing.